src/temp/Explore/ExploreButton.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { ExploreIcon } from '../icons/ExploreIcon';
+import style from './Explore.module.scss';
+import InfoIcon from '../icons/InfoIcon';
+import { StyledSwitch } from './CustomSwitch';
+import { SmallSlider } from './CustomSlider';
+import PlayIcon from '../icons/PlayIcon';
+import { useTranslation } from 'next-i18next';
+import CustomButton from './CustomButton';
+
+interface ExploreButtonProps {
+  label: string | string[];
+  isOpen: boolean;
+  startYear: number;
+  endYear: number;
+}
+
+interface EcosystemOptionProps {
+  infoIcon: React.ReactNode;
+  label: string;
+  switchComponent: React.ReactNode;
+}
+
+interface YearRangeSliderProps {
+  startYear: number;
+  endYear: number;
+}
+
+export const EcosystemOption = ({
+  infoIcon,
+  label,
+  switchComponent,
+}: EcosystemOptionProps) => {
+  return (
+    <>
+      <div className={style.ecosystemMainContainer}>
+        <div className={style.ecosystemContainer}>
+          <div className={style.infoIconConatiner}>{infoIcon}</div>
+          <div>{label}</div>
+        </div>
+        <div className={style.switchContainer}>{switchComponent}</div>
+      </div>
+    </>
+  );
+};
```

Comment:
`EcosystemOption` - this name isn't relevant, not sure how ecosystem fits in. `MapLayerToggle` probably makes more sense, as that's the functionality these toggles provide.